rclrs/src/lib.rs

```diff
@@ -148,6 +153,82 @@ pub fn create_node_builder(context: &Context, node_name: &str) -> NodeBuilder {
     Node::builder(context, node_name)
 }
 
+/// Extract non-ROS arguments from program's input arguments.
+///
+/// `args` is expected to be input arguments of the program (passed via [`env::args()`])
+/// According to rcl documentation ROS arguments are between `--ros-args` and `--` arguments.
+/// Everything beside is considered as non-ROS and left unparsed. Extracted non-ROS args are
+/// returned in order as they appear (see example below).
```

Comment:
```suggestion
/// `args` is expected to be the input arguments of the program (passed via [`env::args()`]).
/// According to rcl documentation, ROS arguments are between `--ros-args` and `--` arguments.
/// Everything else is considered as non-ROS and left unparsed. Extracted non-ROS args are
/// returned in the order that they appear (see example below).
```

Minor change, looks a bit more natural.